app/src/main/res/xml/appfilter.xml

```diff
@@ -1121,247 +1121,487 @@
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Deep_purple}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Indigo}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Blue}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Light_blue}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Cyan}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Teal}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Green}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Light_green}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Lime}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Yellow}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Amber}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Orange}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Deep_orange}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Brown}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Blue_grey}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Grey_black}" drawable="simplecalculator"/>
+    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Pink}" drawable="simplecamera"/>
```

Comment:
shouldn't these use `simplecalculator` instead of `simplecamera`

Reply:
They should, vim got a bit overzealous with the search and replace.